tests/conftest.py

```diff
@@ -683,6 +683,16 @@ def od_detections(od_detection_dataset):
     return learner.predict_dl(od_detection_dataset.test_dl, threshold=0)
 
 
+@pytest.fixture(scope="session")
+def coco_sample_path(tmp_session) -> str:
+    """ Returns the path to a coco-formatted annotation. """
+    return os.path.abspath(
+        os.path.join(
+            os.path.dirname(__file__), os.path.pardir, "data", "misc", "coco", "coco_sample.json"
```

Comment:
really really minor, but i think we've been using this convention elsewhere:
```
os.path.dirname(__file__), os.path.pardir, "data/misc/coco/coco_sample.json"
```

Reply:
In that case, maybe we leave my code as is, since using os.path.join seems like the right thing to do rather than "/"s ?